src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -197,103 +223,45 @@ public function frontendDevDisable(string $siteDir = 'default', array $opts = ['
 
     /**
      * Refreshes a development environment based upon the Drupal version.
-     *
-     * @param string $siteName
-     *   The Drupal site name.
-     * @param bool $skipLandoStart
-     *   If TRUE, skip starting Lando.
      */
-    protected function devRefreshDrupal(string $siteName = 'default', bool $skipLandoStart = false): Result
-    {
+    protected function devRefreshDrupal(
+        LocalDevEnvironmentTypes $environmentType,
+        string $siteName = 'default',
+        bool $startLocalEnv = false,
+    ): Result {
         $this->io()->title('development environment refresh. 🦄✨');
         $result = $this->taskComposerInstall()->run();
-        if (!$skipLandoStart) {
-            $result = $this->taskExec('lando')->arg('start')->run();
+        if ($startLocalEnv) {
```

Comment:
Should everything in this `if` statement be its own method?

Reply:
FWIW, I started working on this and I ended up feeling like I just traded one mess for another. [Leaving this patch here for posterity](https://gist.github.com/markdorison/1f477c1edb22a108a2b165c89b67f050) but I abandoned this approach for now.